### DISCUSSION TITLE
Is BasicSamples module removed out of Serene project ?

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#5868] [Help] [answered]
![image](https://user-images.githubusercontent.com/2279508/126020814-1feb3737-3268-48fc-b1d7-89fe21039e97.png)

![image](https://user-images.githubusercontent.com/2279508/126020799-0392847d-2ed6-4741-9524-216c8ab60985.png)

![image](https://user-images.githubusercontent.com/2279508/126020817-418e793d-289f-4373-a90f-687a0293fe99.png)

**Comment by VictorTomaili** [comment, marked as answer]
https://github.com/serenity-is/common-features

**Reply by minhhungit** [reply]
thank you

**Reply by VictorTomaili** [reply]
your welcome 😄
